ge/views_utils.py

```diff
@@ -906,3 +910,23 @@ def download_zip_file() -> HttpResponse:
     # Attach it to the response and return it.
     response["Content-Disposition"] = f"attachment; filename={zip_filename}"
     return response
+
+
+def add_border_formatting(ws: Worksheet) -> Worksheet:
```

Comment:
@ztucker4 Since this function has a return type of `Worksheet`, there should be a `return ws` at the end.  I know, in Python the `return` is not strictly needed since this is modifying an object's attributes... but it's being called as if something should be returned (`gifts_ws = add_border_formatting(gifts_ws)`), so it's better to be explicit.

Compare with `sum_col()`, which has a return type of `None`, does not have an explicit `return`, and is called like `sum_col(gifts_ws, col)`.  This is consistent and the usage makes sense.  Since the intent of `add_border_formatting()` is to update the given worksheet, similar to `sum_col()`, I think that `add_border_formatting()` should have a return type of `None` and no `return` statement, and should be called like `add_border_formatting(gifts_ws)`.

Double-check the Excel output to be sure these changes don't break anything, and please let me know if this doesn't make sense... I had to refresh my memory on whether Python does ever pass by reference, and what happens in different scenarios.

Thanks --Andy